src/State/Users/Db.ts

```diff
@@ -0,0 +1,147 @@
+import { HexKey } from "Nostr";
+import { db as idb } from "Db";
+
+import { UsersDb, MetadataCache, setUsers } from "State/Users";
+import store from "State/Store";
+
+class IndexedDb implements UsersDb {
+  isAvailable() {
+    if ("indexedDB" in window) {
+      return new Promise<boolean>((resolve) => {
+        const req = window.indexedDB.open("test", 1)
```

Comment:
Might as well try to open the SnortDb so we dont have a random "test" db be there forever